bullet_server/Impulse impulse
---
bool success
